packages/react-storage/src/components/StorageBrowser/views/LocationActionView/UploadView/useUploadView.ts

```diff
@@ -18,7 +18,7 @@ export const useUploadView = (
   const [{ files, location }, dispatchStoreAction] = useStore();
   const { current, key } = location;
 
-  const [isOverwriteEnabled, setOverwriteEnabled] = React.useState(
+  const [isOverwritingEnabled, setObjectOverwriteEnabled] = React.useState(
```

Comment:
Prefer to align state dispatch name with state value, e.g. `setIsOverwritingEnabled`

Reply:
Oops, missed that when I re-updated the naming